sass/components/_dialog.scss

```diff
@@ -132,6 +132,20 @@
             }
         }
     }
+    .info-footnote {
+        @include themify($themes) {
+            color: themed('ChartModeText') !important;
+        }
+        line-height: 18px;
+        font-size: 12px;
+        font-weight: 400;
```

Comment:
font-weight: 400 (aka normal) is default, it can be removed in case if it's not overwritten something. If it's for overwritten, please, ignore my comment.